lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -539,9 +539,10 @@ sub _generate_jobs {
             }
 
             # add properties from dedicated database columns to settings
-            $settings{TEST}    = $job_template->test_suite->name;
-            $settings{MACHINE} = $job_template->machine->name;
-            $settings{BACKEND} = $job_template->machine->backend;
+            $settings{TEST}     = $job_template->test_suite->name;
+            $settings{MACHINE}  = $job_template->machine->name;
+            $settings{BACKEND}  = $job_template->machine->backend;
+            $settings{SCENARIO} = $job_template->name if $settings{SCENARIO};
```

Comment:
doesn't this need to be "JOB_TEMPLATE_NAME" as well?

Reply:
I don't understand why we need a setting variable anyway? Why is setting NAME not good enough right here?

Reply:
the name is auto-generated from all the different entities so we need an override for the job_template_name

Reply:
yeah, but why does it have to be a setting? It just needs to be used when calculcating the name, but isn't that the same spot?

Reply:
I guess it's ok if we can change this later easily. I consider a more critical part the database schema changes which I think you agree with.